api.py

```diff
@@ -1,11 +1,45 @@
-
 from flask import Flask
+from alembic import op
+import sqlalchemy as sa
+import psycopg2
 
 app = Flask(__name__)
-
+con = psycopg2.connect(database="db_api_teste", user="root", password="root", host="localhost", port="5432")
+cur = con.cursor()
 @app.route('/')
 def hello():
```

Comment:
```suggestion
def ola():
```
Para manter a consistência com o resto do código